ivy/functional/backends/torch/statistical.py

```diff
@@ -293,3 +293,24 @@ def einsum(
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
     return torch.einsum(equation, *operands)
+
+
+def hann_window(
```

Comment:
It seems you have implemented for torch already in statistical so can safely remove this